mono/mini/iltests.il

```diff
@@ -2896,4 +2896,50 @@ END:
     IL_0018:  ldloc.0
 	ret
 	}
+
+  .method private hidebysig static void  fail_inline() cil managed
+  {
+    // Code size       9 (0x9)
+    .maxstack  8
+    IL_0000:  ldc.i4.s   16
+    IL_0002:  conv.u
+    IL_0003:  ldc.i4.1
+    IL_0004:  mul.ovf.un
+    IL_0005:  localloc
+    IL_0007:  pop
+    IL_0008:  ret
+  }
+
+  .method private hidebysig static int32
+          always_inline(int32 op) cil managed aggressiveinlining
```

Comment:
`iltests.il (2914, 0) : Error : irrecoverable syntax error`

looks like 2017-08 is missing the `aggressiveinlining` ilasm support: https://github.com/mono/mono/commit/0290d19c8391ff5a16fe32a073221f34245afc5d